packages/notifier/.eslintrc-jessie.js

```diff
@@ -0,0 +1,7 @@
+/* global module */
```

Comment:
Why are these appearing as part of this PR's diffs? Aren't they already part of notifier-now-master?

rebase?

Reply:
Ah! I forgot! Don't rebase! Sorry for contradicting myself.